src/sql/parts/connection/common/connectionManagementService.ts

```diff
@@ -659,7 +660,7 @@ export class ConnectionManagementService implements IConnectionManagementService
 	}
 
 	public getActiveConnections(): ConnectionProfile[] {
-		return this._connectionStore.getActiveConnections();
```

Comment:
this method is not used by anything else? just wondering if the change would affect other part of UI. I know the filtering connection tree uses this method

Reply:
It's not used elsewhere right now and would actually give wrong results if it were used, since there are some bugs with how the connection store tracks active connections. I think it's called once from the filtering connection tree code, but that part of the code never actually gets called as it is